sdk/core/azure-core/test/e2e/azure_core_with_curl_bodyBuffer.cpp

```diff
@@ -72,18 +72,41 @@ int main()
   return 0;
 }
 
-#if defined(AZ_PLATFORM_POSIX)
 void doFileRequest(Context const& context, HttpPipeline& pipeline)
 {
 
   Azure::Core::Http::Url host("https://httpbin.org/put");
   cout << "Creating a Put From File request to" << endl
        << "Host: " << host.GetAbsoluteUrl() << endl;
 
+#if defined(AZ_PLATFORM_POSIX)
   // Open a file that contains: {{"key":"value"}, {"key2":"value2"}, {"key3":"value3"}}
   int fd = open("/home/vivazqu/workspace/a", O_RDONLY);
+  if (fd < 0)
+  {
+    throw std::invalid_argument("Fail to open the requested file");
+  }
   // Create Stream from file starting with offset 18 to 100
   auto requestBodyStream = FileBodyStream(fd, 18, 100);
+#elif defined(AZ_PLATFORM_WINDOWS)
+  // NOTE: To run the sample: Create folder 'home' on main hard drive (like C:/) and then add a file
+  // `a` in there
+  //
```

Comment:
Running tests and samples shouldn't require manual artifact creation on the machine. Especially not in directories external to the repo.

The test should create whatever file or directory it needs within the sub-folder inside the repo and use it.